R/colors_race.R

```diff
@@ -8,21 +8,31 @@
 #'
 #' @noRd
 make_race_palette <- function(white, black, hispanic, asian, other) {
+    if (missing(white) & missing(black) & missing(hispanic) & missing(asian) & missing(other)) {
+      stop("You must specify at least one race category", call. = FALSE)
+    }
+
+    passed <- unlist(as.list(match.call())[-1]) # vector of args actually passed
+
     race_palette <- fetch_pal("race")
-    if (!missing(white)) { names(race_palette)[1] <- white }
```

Comment:
I think this section is going to need to be maintained in some way, although I'm not sure on the sequencing. But the goal would be that, in cases where people do have default values, the function remains functional, even without specification.

Reply:
In the case where the dataframe already has the default values, the code has been updated for the function to return the default race palette.